src/main/kotlin/com/tumugin/aisu/Application.kt

```diff
@@ -22,11 +22,11 @@ fun main() {
 
 fun createKtorModule(koin: Koin): Application.() -> Unit {
   val ktorModule: Application.() -> Unit = {
+    configureAuthentication(koin)
```

Comment:
Needs some fixes(Authentication plugin must be installed before routing plugin install)